frontend/src/tabs/authorship.pug

```diff
@@ -77,6 +77,13 @@
               span(v-bind:title="getFileTypeBlankLineInfo(fileType)")
                 span {{ fileType }}&nbsp;{{ fileTypeLinesObj[fileType] }}&nbsp;
                 span ({{ fileTypeLinesObj[fileType] - fileTypeBlankLinesObj[fileType] }})&nbsp;
+          label.binary-fileType(v-if="binaryFilesCount > 0")
+            input.mui-checkbox--fileType(type="checkbox", v-bind:checked="isBinaryFilesChecked",
```

Comment:
if it is independent now, should be able to directly bind the value to a vue model?

Reply:
Sure, no problem